Main.html

```diff
@@ -139,8 +139,7 @@ <h1 id="title" class="text-center">FreeCodeCamp Survey Form</h1>
               value="open-source-community"
               type="checkbox"
               class="input-checkbox"
-            />Open Source Community</label
```

Comment:
-it would be great to omit unnecessary breaks and new lines in the code.

Reply:
- [ ] Great job on giving feedback, but please use polite ways to communicate when giving feedback and also provide a reason `why` it needs a change. For further reference [check this](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md#how-to-use-github-features)

Reply:
Good job with the code,  documenting your code will make it look cleaner.